vpc_scripts/source_posix_base.vpc

```diff
@@ -37,7 +37,7 @@ $Configuration
 
 	$Compiler
 	{
-		$AdditionalIncludeDirectories		"$SRCDIR\common;$SRCDIR\public;$SRCDIR\public\tier0;$SRCDIR\public\tier1"
+		$AdditionalIncludeDirectories		"$SRCDIR\common;$SRCDIR\public;$SRCDIR\public\tier0;$SRCDIR\public\tier1;/usr/include/freetype2/"
```

Comment:
System-dependent changes will not be merged

Reply:
fixed